Sources/ProjectSpec/TargetSource.swift

```diff
@@ -32,17 +33,52 @@ public struct TargetSource: Equatable {
         case sources
         case headers
         case resources
+        case copyFiles
         case none
 
         public var buildPhase: xcproj.BuildPhase? {
             switch self {
             case .sources: return .sources
             case .headers: return .headers
             case .resources: return .resources
+            case .copyFiles: return .copyFiles
             case .none: return nil
             }
         }
     }
+    
+    public struct CopyFiles: Equatable, Hashable {
```

Comment:
Thinking about this, maybe this should be called `CopyFilesBuildPhaseSettings`. I know it's longer, but I think it makes it clear what it's for.